guide/plans/plan-model.mdx

```diff
@@ -57,14 +57,23 @@ applied to the Plan regarding the number of days offered.
   upgrade or downgrade, the trial period of the new Plan **is not applied**.
 </Warning>
 
+The free trial only applies to the subscription fee. Any pay in advance charges
+will be invoiced and collected immediately after an event is received, even during
+the trial period.
```

Comment:
Any events received during the trial period will be considered at the end of billing aggregation. Pay-in-advance charges will be invoiced and collected immediately after an event is received.

wdyt of this one ⏫? cc. @julienbourdeau

Reply:
edited